.gitlab/ci.yml

```diff
@@ -116,6 +119,8 @@ tarball ubuntu-16.04:
         - apt-get install -yqq build-essential libgmp-dev zlib1g-dev
         # build tools for ctags
         - apt-get install -yqq dh-autoreconf pkg-config
+        # build tools for OpenLDAP
+        - apt install -yqq groff groff-base
```

Comment:
Based on OpenLDAP's CI script, only `groff-base` seems to be required:
https://git.openldap.org/openldap/openldap/-/blob/master/.gitlab-ci.yml

Reply:
I removed groff.